app/mailboxes/ticket_mailbox.rb

```diff
@@ -4,19 +4,7 @@
 # <<process>> method is mandatory
 
 class TicketMailbox < ApplicationMailbox
-  before_processing :ensure_user
-
   def process
     Ticket.create!(subject: mail.subject, body: mail.body, user:)
   end
-
-  private
-
-  def user
-    User.find_by(email: mail.from)
-  end
-
-  def ensure_user
```

Comment:
Why is it that you don't need this kind of validations here?

Reply:
this is because both Action Mailbox files need the same validation. For that reason, I move it into Application Mailbox.

Reply:
```
class ApplicationMailbox < ActionMailbox::Base
  before_processing :ensure_user_presence.  <<<
```
  this callback do the magic